cache/.htaccess

```diff
@@ -1 +0,0 @@
-Deny from all
```

Comment:
We need to make sure this is deleted and put it back on master branch as well.

Reply:
Why did we do that?